ext/dispatcher.go

```diff
@@ -243,13 +242,13 @@ func (d *Dispatcher) ProcessUpdate(b *gotgbot.Bot, update *gotgbot.Update, data
 			// If a panic handler is defined, handle the error.
 			if d.Panic != nil {
 				d.Panic(b, ctx, r)
-				return
 
-			} else {
-				// Otherwise, create an error from the panic, and return it.
-				err = fmt.Errorf("%w: %v\n%s", ErrPanicRecovered, r, cleanedStack())
 				return
 			}
+			// Otherwise, create an error from the panic, and return it.
+			err = fmt.Errorf("%w: %v\n%s", ErrPanicRecovered, r, cleanedStack())
```

Comment:
This looks like a logic change. Previously, the Panic didn't set the error (as the idea was that it had been handled).

Is this intentional? If yes, what is the reasoning behind it?

Reply:
I've just changed 
```
if something {
    // some code
    return
} else {
    // some code
    return
}
```
by
```
if something {
    // some code
    return
}
// some code
return
```

AFAIK, no logic change.

Reply:
Ah - yes, my bad! Misread the git diff and didnt see that the second return wasnt removed. You're entirely right!